test/jdk/sun/security/pkcs12/KeytoolOpensslInteropTest.java

```diff
@@ -67,18 +67,18 @@
 public class KeytoolOpensslInteropTest {
 
     public static void main(String[] args) throws Throwable {
-        String opensslPath = OpensslArtifactFetcher.getOpenssl1dot1dotStar();
+        String opensslPath = OpensslArtifactFetcher.getOpensslPath();
         if (opensslPath != null) {
-            // if preferred version of openssl is available perform all
+            // if the current version of openssl is available, perform all
             // keytool <-> openssl interop tests
             generateInitialKeystores(opensslPath);
             testWithJavaCommands();
             testWithOpensslCommands(opensslPath);
         } else {
-            // since preferred version of openssl is not available skip all
+            // since the current version of openssl is not available, skip all
             // openssl command dependent tests with a warning
-            System.out.println("\n\u001B[31mWarning: Can't find openssl "
-                    + "(version 1.1.*) binary on this machine, please install"
+            System.out.println("\n\u001B[31mWarning: Can't find the current version "
```

Comment:
It will be better to update this to print `OpensslArtifactFetcher.OPENSSL_BUNDLE_VERSION` with the corresponding `getTestOpenSSLBundleVersion` routine in `OpensslArtifactFetcher`. The current version verbiage is confusing.

Reply:
Ok thanks, updated with suggestions